dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -261,7 +140,7 @@ struct ha_data_t {
     bit<16> dp_channel_src_port_max;
 
     // HA packet/flow state
-    dash_ha_flow_sync_state_t flow_sync_state;
+    dash_flow_state_t flow_sync_state;
```

Comment:
since you renamed the struct, will be better to also rename the field

Reply:
Rename it to `ha_flow_state`.